src/main/java/com/shapesecurity/shift/es2018/fuzzer/Fuzzer.java

```diff
@@ -894,12 +894,12 @@ private static double randomNumber(@Nonnull GenCtx ctx) {
 
     @Nonnull
     private static ObjectBinding randomObjectBinding(@Nonnull GenCtx ctx, int depth) {
-        return new ObjectBinding(many(Fuzzer::randomBindingProperty).apply(ctx, depth - 1));
+        return new ObjectBinding(many(Fuzzer::randomBindingProperty).apply(ctx, depth - 1), Maybe.empty()); // TODO fuzz rest
```

Comment:
Let's address these TODOs. Shouldn't be too hard; look at fuzzing any other Maybe.